iOS/MobileStarter/react-app/src/ProjectPicker.tsx

```diff
@@ -0,0 +1,143 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import { HorizontalPicker, useIsMountedRef } from "@itwin/mobile-ui-react";
+import { Project, ProjectsAccessClient, ProjectsQueryArg, ProjectsQueryFunction, ProjectsSearchableProperty, ProjectsSource } from "@itwin/projects-client";
+import { ProgressCallback } from "@itwin/core-frontend/lib/cjs/request/Request";
+import { IModelApp } from "@itwin/core-frontend";
+import { LoadingSpinner } from "@itwin/core-react";
+import { SearchControl, i18n, presentError, ButtonProps, HubScreenButton, HubScreenButtonListProps, HubScreenButtonList } from "./Exports";
+
+// Get all the projects that this user has access to, sorted by most recent use.
```

Comment:
They're only sorted by most recent use when source is Recent.
It only gets "all" when source is All. (And even then it's only 100 arbitrary ones if the user has access to more than 100.) So this comment needs to be rewritten entirely, I think.

Reply:
I removed the comment